src/libs/elektra/keytest.c

```diff
@@ -446,8 +444,8 @@ printf("key1 and key2 have different values\n");
 
  kdbGet(handle, ks, base);
 
-// we are interested only in name and/or value
-interests=(KEY_NAME | KEY_VALUE);
+// we are interested only in meta and/or value
```

Comment:
That name is handled by "1" should be documented or better: we remove keyCompare and keyCompareMeta as they are hardly used anyway.

Reply:
Thank you for the feedback!
As you've probably already seen, I removed the keyCompare function and moved keyCompareMeta to the specload-plugin, because that was the only place were it was used.